go.mod

```diff
@@ -1,7 +1,8 @@
 module github.com/sjeanpierre/passenger-datadog-monitor
 
 require (
-	github.com/PagerDuty/godspeed v0.0.0-20180224001232-122876cde329
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/theckman/godspeed v1.1.0
```

Comment:
This supersedes the PagerDuty release: https://github.com/PagerDuty/godspeed#deprecation-notice